### PR TITLE
KOA 4746 document state of stories

### PR DESCRIPTION
**Pull request by k0nserv**
Document state of stories.

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/master/Example/Backpack%20Screenshot/Screenshots.swift)
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/master/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Example/Backpack/NavigationData.swift [hunk 1]
```diff
@@ -16,6 +16,29 @@
...
+ * The implementation is based on Swift's
+ * [Result Builders](https://www.swiftbysundell.com/articles/deep-dive-into-swift-function-builders/),
+ * the very same that powers Swift UI. The main entry point for this is the function ``makeApp``.
+ *
+ * Prior to this system, we used storyboards to describe the navigation and structure of the app. We are in the process
+ * of migrating from that system to the stories format.
+ *
+ * The following PRs contain examples of this:
+ *
+ * * [#940](https://github.com/Skyscanner/backpack-ios/pull/940) _d8de142_
```
Comment (georgegillams) [inline Example/Backpack/NavigationData.swift:36]: Is `_d8de142_` meant to be here?

Comment (k0nserv) [reply Example/Backpack/NavigationData.swift:36]: Yeah, it's the commit in question. To make sure the change can still be found if we migrate from GitHub

File: Example/Backpack/NavigationData.swift [hunk 2]
```diff
@@ -16,6 +16,29 @@
...
+ * * [#937](https://github.com/Skyscanner/backpack-ios/pull/937)
+ *
+ * In ``appStructure`` below, any code that still uses the function ``loadStoryboard`` still needs to be migrated.
+ * When the migration is done we should be left with no storyboards.
```
Comment (georgegillams) [inline Example/Backpack/NavigationData.swift:40]: I thought we decided that storyboards were ok for laying out examples, but not for navigation. Maybe we could make this clearer here?

Comment (k0nserv) [reply Example/Backpack/NavigationData.swift:40]: Eh yeah, you're right. Can you make that change perhaps? I'm working on a different branch atm

File: Example/Backpack/NavigationData.swift (hunk 1 above)
Comment (georgegillams) [reply Example/Backpack/NavigationData.swift:36]: So maybe the other PR should have the hash too?

Comment (k0nserv) [reply Example/Backpack/NavigationData.swift:36]: Yeah good shout

**Review by georgegillams — APPROVED** [review]

**Pull request merged**